deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -127,9 +132,9 @@ spec:
     - [AWS](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-aws.md)
 
     ### Reporting issues
-    Issues with this distribution of WMCO can be opened against the [Windows Container Bugzilla component](https://bugzilla.redhat.com/buglist.cgi?product=OpenShift%20Container%20Platform&component=Windows%20Containers&resolution=---).
+    Support for this distribution of WMCO requires a subscription. Support should be requested through the Red Hat Customer Portal.
     Please read through the [troubleshooting document](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/TROUBLESHOOTING.md)
```

Comment:
Troubleshooting link needs to be replaced with a link to OpenShift docs.

Reply:
https://docs.openshift.com/container-platform/4.6/support/troubleshooting/troubleshooting-windows-container-workload-issues.html